### PR TITLE
Pyrenew tutorial review

### PR DESCRIPTION
**Pull request by sbidari**
Some ideas for improving the documentation

1. Introduction to renewal model framework and some of the model components implemented in the package. The model equations/information from [here](https://github.com/CDCgov/multisignal-epi-inference/blob/main/model/README.md) and [here](https://github.com/CDCgov/multisignal-epi-inference/blob/main/model/equations.md) can perhaps be moved to the documentation
2. Include installation instructions in the beginning of the documentation either in a separate section or within the getting started section.
3. I am not entirely sure what is the difference between the `HospitalAdmissionsModel` in Pyrenew demo and the one in Fitting a Hospital Admissions-only model. Perhaps clarify what the two tutorials are trying to convey. 
4. If I understand correctly there are two different model implementation - Hospital Admissions and Reproduction Number Renewal Infections. But it isn't clear from the tutorial, suggesting reorganizing the tutorials to make these two model instances examples clear.
5. Change the order of tutorial and reference in the documentation

I can make most of these changes (might need some help on 1) if we think these would be useful.

### CODE REVIEW

**Comment by damonbayer** [comment]
> Some ideas for improving the documentation
> 
> 1. Introduction to renewal model framework and some of the model components implemented in the package. The model equations/information from [here](https://github.com/CDCgov/multisignal-epi-inference/blob/main/model/README.md) and [here](https://github.com/CDCgov/multisignal-epi-inference/blob/main/model/equations.md) can perhaps be moved to the documentation
> 2. Include installation instructions in the beginning of the documentation either in a separate section or within the getting started section.
> 3. I am not entirely sure what is the difference between the `HospitalAdmissionsModel` in Pyrenew demo and the one in Fitting a Hospital Admissions-only model. Perhaps clarify what the two tutorials are trying to convey.
> 4. If I understand correctly there are two different model implementation - Hospital Admissions and Reproduction Number Renewal Infections. But it isn't clear from the tutorial, suggesting reorganizing the tutorials to make these two model instances examples clear.
> 5. Change the order of tutorial and reference in the documentation
> 
> I can make most of these changes (might need some help on 1) if we think these would be useful.

1. This will be covered (at least partially) by https://github.com/CDCgov/multisignal-epi-inference/pull/197
2. I think installation instructions can go in both the main website page and the getting started tutorial. Note that installation instructions are also here, which is not on the website: https://github.com/CDCgov/multisignal-epi-inference/tree/main/model#readme. This may need to be updated.
3. Agreed that these are confusing/redundant. I would lean toward merging the `PyRenew Demo` and `Fitting a Hospital Admissions-only Model` tutorials. Probably by keeping `Fitting a Hospital Admissions-only Model` mostly as is, but bringing over anything useful from `PyRenew Demo` that it is missing.
4. That's correct. I agree making this more obvious would be useful. We provide two pre-built models: `RtInfectionsRenewalModel` and `HospitalAdmissionsModel`. `HospitalAdmissionsModel` uses `RtInfectionsRenewalModel` and builds upon it.
5.  Sounds good.

**Comment by sbidari** [comment]
> 4. That's correct. I agree making this more obvious would be useful. We provide two pre-built models: `RtInfectionsRenewalModel` and `HospitalAdmissionsModel`. `HospitalAdmissionsModel` uses `RtInfectionsRenewalModel` and builds upon it.

I liked the diagram here: https://github.com/CDCgov/multisignal-epi-inference/tree/main/model#readme and can be effective visualization to show `HospitalAdmissionsModel` uses `RtInfectionsRenewalModel` and builds upon it.

**Comment by damonbayer** [comment]
@sbidari Would it be appropriate to also address [these comments](https://teams.microsoft.com/l/message/19:7c1e84b1ca6d48559ee487f9477ae57b@thread.tacv2/1719603439343?tenantId=9ce70869-60db-44fd-abe8-d2767077fc8f&groupId=4f34eba7-8fcb-4a26-9b34-a434ea777f0c&parentMessageId=1719603439343&teamName=CFA-Predict&channelName=Proj%20-%20Multisignal%20Renewal&createdTime=1719603439343) from @cherz4 in this PR? If not, please create additional issues related to the points discussed on Teams.

**Comment by sbidari** [comment]
Yes, will include more instructions for poetry installation.

**Comment by kgostic** [comment]
Re: 2 - I think this would be a good place to add some installation instructions:

https://github.com/CDCgov/multisignal-epi-inference/blob/da3fae6a73e5b16e83cfaf986860d33fa39730ac/model/docs/getting_started.qmd#L8

Specifically, I'd suggest a **Prerequisites** section before **The fundamentals** that reads something like this:

## Prerequisites
This tutorial assumes some pre-existing knowledge of infectious disease dynamics and Python programming. Before you dive in, we recommend:

* Installing [Python3](link-to-installation-instructions)
* Familiarity with installing and loading modules in python, and with virtual environment management (we recommend [poetry](link-to-tutorial])) 
* Familiarity with the concept of a [class](link-to-docs) and [metaclass](link-to-docs) in python
* Familiarity with Bayesian inference, and a working understanding of MCMC methods used to fit Bayesian models to data (link to resources like statistical rethinking?)

**Comment by kgostic** [comment]
Minor: I think this phrasing makes it sound like we're taking the ratio $\frac{I_t}{I_{t-s}}$, rather than considering all previous timesteps in the denominator. 

```
The pyrenew package models the real-time reproductive number $R_t$, the ratio of new infections at 
time $t$ to previous infections at some time $t-s$, as a renewal process model
```

https://github.com/CDCgov/multisignal-epi-inference/blob/da3fae6a73e5b16e83cfaf986860d33fa39730ac/model/docs/getting_started.qmd#L65C1-L65C236

**Comment by sbidari** [comment]
Changes to the tutorial/documentation in this PR:

- added a page with Prerequisite and installation information
- merge/remove pyrenew demo
- rename 'example_with_dataset' to 'hospital_admissions_model'
- demonstrate extraction of samples from the fitted model

File: model/docs/getting_started.qmd [hunk 1]
```diff
@@ -33,340 +57,3 @@ class MyRandVar(RandomVariable):
 Thus, when a `Model` samples from `MyRandVar`, it could be either adding random variables to be estimated (first case) or just retrieving some quantity needed for other calculations (second case.)
 
 The `Model` metaclass provides basic functionality for estimating and simulation. Like `RandomVariable`, the `Model` metaclass has a `sample()` method that defines the model structure. Ultimately, models can be nested (or inherited), providing a straightforward way to add layers of complexity.
```
Comment (damonbayer) [inline model/docs/getting_started.qmd:59]: Maybe after this, we can point to one of our models or tutorials as an example / "next step?"

**Review by damonbayer — COMMENTED** [review]
This PR does not address tests, which is fine. However, we should change the title of the PR and make sure we still have an open issue for the tests.

File: model/docs/basic_renewal_model.qmd
Comment (damonbayer) [inline model/docs/basic_renewal_model.qmd:1]: I think this is missing from the website navigation

File: model/docs/hospital_admissions_model.qmd
Comment (damonbayer) [inline model/docs/hospital_admissions_model.qmd:1]: This is also missing from the website navigation

**Review by damonbayer — CHANGES_REQUESTED** [review]
Please add the new/renamed tutorials to the website navigation.

File: model/docs/getting_started.qmd [hunk 2]
```diff
@@ -10,7 +10,7 @@ engine: jupyter
 
 This tutorial assumes some pre-existing knowledge of infectious disease dynamics and Python programming. Before you dive in, we recommend:
 
- - Installing Python3 (use tools like [pyenv](https://realpython.com/intro-to-pyenv/) or [compile and install](https://ubuntuhandbook.org/index.php/2023/05/install-python-3-12-ubuntu/) from the [source page](https://www.python.org/downloads/))
+ - Verifying you have Python3.12 installed. If using wsl on VAP or GFE, the default python interpretor is likely python3.10 but pyrenew requires python3.12 (use tools like [pyenv](https://realpython.com/intro-to-pyenv/) or [compile and install](https://ubuntuhandbook.org/index.php/2023/05/install-python-3-12-ubuntu/) from the [release page](https://www.python.org/downloads/))
```
Comment (dylanhmorris) [inline model/docs/getting_started.qmd:13]: Suggest avoiding language that's specific to internal users in our external docs

Comment (damonbayer) [reply model/docs/getting_started.qmd:13]: I think the previous version (that doesn't mention 3.12 specifically) would be acceptable.

Comment (sbidari) [reply model/docs/getting_started.qmd:13]: Got it!

**Review by damonbayer — APPROVED** [review]
I think there is still plenty of room for improvement in the tutorials, but this is a big step in the right direction. Thanks @sbidari!

**Pull request merged**